nixos/modules/services/networking/wg-netmanager.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.wg-netmanager;
+in
+{
+
+  options = {
+    services.wg-netmanager = {
+      enable = mkEnableOption "Wireguard network manager";
+      package = mkOption {
```

Comment:
This `package` option seems to have no effect, as you use `${pkgs.wg-netmanager}` directly below.

If there is no alternative implementation I would just drop it.

Otherwise if you want to keep it and have an effect. You should change `${pkgs.wg-netmanager}` `${cfg.package}` in the service definition.

Reply:
Actually I have no clue, what this all means. I have just copy-and-paste'd it together. Hope the latest patch is OK